### DISCUSSION TITLE
Realm migration issue (One or more errors occurred. (A Ruleset object already exists with primary key property ShortName == '"osu"'))

### DISCUSSION THREAD

**Discussion by saturnSolar** [ppy/osu#16656] [Q&A] [answered]
yeah
[attach_me.zip](https://github.com/ppy/osu/files/7951969/attach_me.zip)

**Comment by frenzibyte** [comment, marked as answer]
Your database has beatmaps from the following missing rulesets (rush, tau, karaoke!, solosu). Overwrite the existing `client.realm` file in your osu!(lazer) folder with the one in this archive: [client.realm.zip](https://github.com/ppy/osu/files/7952112/client.realm.zip)

**Reply by saturnSolar** [reply]
alright it works tysm
